### ISSUE TITLE
Add support for `pixi add -f requirements.txt`

### ISSUE THREAD

**Issue by rachfop** [prefix-dev/pixi#2206]
### Problem description

I would like to take existing Python projects which normally contains a `requirements.txt` file and install the libraries with `pixi`. 

A command like:

```
pixi add --file <path to requirements.txt>
```
Would be helpful.

**Comment by ruben-arts** [comment]
I like the idea, I think it could be pretty beneficial for moving to `pixi`. 

I would like to get the design a little more written out. 

It should be designed to support more then only `pypi` `requirements.txt`. Some ideas I would have are:

- `requirements.txt` for conda packages (CES) 
- `environment.yml`, we can already `pixi init --import` that but it seems like a good idea to also allow you to do that on existing projects.
- `conda-lock` files
And possibly more that aren't on top of my mind.

We already support `pixi init --import` we could reuse that wording by making it `pixi add --import`.

Could you provide more input on the ideas and implementation details?

**Comment by zbowling** [comment]
I like the idea for dealing with legacy projects but new projects should be encouraged to update to pyproject.toml. requirements.txt is actually an imperative list of instructions that have be read top to bottom because it’s actually a list of args that pip slurps in (and new tools have to emulate pip because you can pip args in there like alternative pypi mirrors) and not actually declarative like the toml file.

**Comment by ruben-arts** [comment]
I agree @zbowling, thus only making it an "import" type function and not a split of information between files. A pixi.lock should only be made if the information is in the manifest.

**Comment by straygar** [comment]
I'd also like support for this in pixi import, when migrating from conda.
```bash
pixi init --import environment.yml
```
currently fails if you have specified:
```
pip:
  - -r requirements.txt
```

in your environment.yaml

**Comment by tdejager** [comment]
@straygar there is a PR but it's not being actively developed: https://github.com/prefix-dev/pixi/pull/2883

**Comment by lucascolley** [comment]
Roadmap for related work: https://github.com/prefix-dev/pixi/issues/4192

**Comment by lucascolley** [comment]
I think we should be able to close this issue on merge of gh-4270. Rather than `pixi add -f requirements.txt`, the command will be `pixi import --format=pypi-txt requirements.txt`.
